### PR TITLE
canary documentation

### PR DESCRIPTION
**Pull request by rouzwelt**
## Motivation
resolves https://github.com/rainlanguage/arb-bot/issues/199

## Checks

By submitting this for review, I'm confirming I've done the following:
- [x] made this PR as small as possible
- [ ] unit-tested any new functionality
- [x] linked any relevant issues or PRs

### CODE REVIEW

File: CANARY.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Canary strat
```
Comment (thedavidmeister) [inline CANARY.md:1]: i don't really understand why this doc is here, in the root of the ob repo

or who it is for?

@rouzwelt originally i asked you to setup a canary, which is already documented in the strat in pubstrats in basically paraphrasing exactly what is here

```

# The canary order does nothing except run on a cooldown.
#
# It is useful only to show that orders clear on some network. If the canary is
# deployed and does not run, it means that there are no solvers monitoring the
# network, or at least not the raindex contract that the canary is deployed on.
#
# The canary is basically a stripped back version of the sampler. It runs but
# doesn't sample anything.
```

i'd expect to see docs that explain what was missing in your understanding when i asked you about it @rouzwelt and probably somewhere related to the infra they are for

**Review by thedavidmeister — CHANGES_REQUESTED** [review]

**Comment by dcatki** [comment]
@rouzwelt whats the status of this?

**Comment by rouzwelt** [comment]
I guess this now can be closed in favor of the canary strats on web app? @thedavidmeister

**Comment by hardyjosh** [comment]
closing this, not required anymore

**Pull request closed without merging**